gio/src/subclass/async_initable.rs

```diff
@@ -0,0 +1,522 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::object::Cast;
+use glib::thread_guard::ThreadGuard;
+use glib::translate::*;
+use glib::Error;
+
+use glib::subclass::prelude::*;
+
+use std::ptr;
+
+use crate::AsyncInitable;
+use crate::AsyncResult;
+use crate::Cancellable;
+
+use std::{future::Future, pin::Pin};
+
+pub trait AsyncInitableImpl: ObjectImpl {
+    #[allow(unused_variables)]
```

Comment:
what are the unused variables?

Reply:
All the arguments to the trait functions were unused, I didn't want to put an underscore in front of them because rust-analyzer copies them when filling in the trait. Since I removed `init_async`, it's no longer needed.